studio/data/projects/project-detail-query.ts

```diff
@@ -26,6 +26,7 @@ export const useProjectDetailQuery = <TData = ProjectDetailData>(
     ({ signal }) => getProjectDetail({ ref }, signal),
     {
       enabled: enabled && typeof ref !== 'undefined',
+      staleTime: 30 * 1000, // 30 seconds
```

Comment:
does this mean the project details keep refreshing every 30 sec?

Reply:
Not exactly, it means it'll only refetch at most once every 30 seconds, instead of at every refetch event (window refocus, etc). It means that when I'm manually setting the cache entry for a project status change, it won't refetch right away and overwrite my changes (there's a route change right after setting the status, which causes a refetch).